src/pages/Vaults/index.tsx

```diff
@@ -4,20 +4,17 @@ import { useTranslation } from 'react-i18next';
 
 import PrimaryColorEllipsisLoader from 'components/PrimaryColorEllipsisLoader';
 import MainContainer from 'parts/MainContainer';
-// import { Grid, GridItem, InfoBox, VaultCard, NewVaultsTable } from 'component-library';
```

Comment:
These changes aren't part of this PR, but this commented out code can be removed as it's being handled in a separate branch now.